cmd/kubelet/app/options/options.go

```diff
@@ -141,6 +141,7 @@ func NewKubeletServer() *KubeletServer {
 			KubeAPIQPS:                     5.0,
 			KubeAPIBurst:                   10,
 			ExperimentalFlannelOverlay:     experimentalFlannelOverlay,
+			StorageConfigDir:               "",
```

Comment:
Why is this option needed in kubelet? It won't provision any volumes, `--storage-config=`` is relevant only to kube-controller-manager